source/quick-start.txt

```diff
@@ -207,6 +207,48 @@ You can use the ``write()`` method to write objects of the following types to Mo
    write(coll, arrow_table)
    write(coll, ndarrays)
 
+You can add an optional argument to ``write()`` called ``exclude_none``.
+This argument is ``False`` by default. When it is set to ``True``, empty
+values will not be written to the database.
+
+The code in the following example writes an Arrow ``Table`` to MongoDB twice: once
+with ``exclude_none`` defaulting to ``False``, and once with it set to ``True``.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: 
+      :language: python
+      :dedent:
+      
+      data_a = [1,2, 3, 4, 5, 6]
+      data_b = [1, 2, 3, None, 5, 6]
```

Comment:
Q: Does `exclude_none` ignore the entire `data_b` array because it contains a single `None` value? If so, might want to reword some things earlier

Reply:
No, it just ignores the one ``None`` value. I will make the example more clear that this is the case